alumnium/drivers/appium_driver.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+
+from appium.webdriver import Remote
+from appium.webdriver.webelement import WebElement
+from appium.webdriver.common.appiumby import AppiumBy as By
+
+from alumnium.accessibility import XCUITestAccessibilityTree
+from alumnium.logutils import get_logger
+
+from .base_driver import BaseDriver
+from .keys import Key
+
+logger = get_logger(__name__)
+
+
+class AppiumDriver(BaseDriver):
+    def __init__(self, driver: Remote):
+        self.driver = driver
+
+    @property
+    def aria_tree(self) -> XCUITestAccessibilityTree:
+        return XCUITestAccessibilityTree(self.driver.page_source)
+
+    def click(self, id: int):
+        self._find_element(id).click()
+
+    def drag_and_drop(self, from_id: int, to_id: int):
+        # TODO: Implement drag and drop functionality
```

Comment:
I'd prefer to land this PR first and then work on implementing other methods with tests.